var/spack/repos/builtin/packages/uncrustify/package.py

```diff
@@ -10,12 +10,30 @@ class Uncrustify(Package):
     """Source Code Beautifier for C, C++, C#, ObjectiveC, Java, and others."""
 
     homepage = "http://uncrustify.sourceforge.net/"
-    url      = "http://downloads.sourceforge.net/project/uncrustify/uncrustify/uncrustify-0.61/uncrustify-0.61.tar.gz"
-
-    version('0.67', sha256='54f15c8ebddef120522db21f38fac1dd3b0a285fbf60a8b71f9e333e96cf6ddc')
+    git      = "https://github.com/uncrustify/uncrustify"
+    url      = "https://sourceforge.net/projects/uncrustify/files/uncrustify/uncrustify-0.69/uncrustify-0.69.tar.gz"
+
+    maintainers = ['gmaurel']
```

Comment:
To make sure: did you check with this user before adding them here? Apologies if they mentioned being interested in maintaining this package somewhere and I missed it.

Reply:
I asked them in uncrustify/uncrustify#3400 and at least @gmaurel pointed out to be the maintainer of the uncrustify package. I am not sure if I understood it well that this user will be maintaining the spack repo in the future.

Otherwise I would be willing to maintain the spack repo in the future too.